src/core/parentClient.ts

```diff
@@ -88,4 +88,29 @@ export class ParentClient extends BaseClient {
     }
     throw new Error("No pupil with specified ID returned");
   }
+    /**
+   * Changes the login password for this parent's account
+   * @param current Current password
+   * @param new New password
+   * @returns whether it was successful or not
+   */
+    async changePassword(this: ParentClient, current: string, newPassword: string): Promise<ChangePasswordResponse>{
+      const formData = new URLSearchParams();
+      formData.append("current", current);
+      formData.append("new", newPassword);
+      formData.append("repeat", newPassword);
+      const headers = new Headers({
```

Comment:
```suggestion
```
I'd prefer this simply as a JSON object passed straight to the headers parameter, to keep it consistent with the rest of the lib.